tests/mixins/test_browsing.py

```diff
@@ -164,9 +165,25 @@ def test_get_song_related_content(self, yt_oauth, sample_video):
 
     def test_get_lyrics(self, config, yt, sample_video):
         playlist = yt.get_watch_playlist(sample_video)
+        # test normal lyrics
         lyrics_song = yt.get_lyrics(playlist["lyrics"])
-        assert lyrics_song["lyrics"] is not None
-        assert lyrics_song["source"] is not None
+        assert lyrics_song is not None
+        assert isinstance(lyrics_song["lyrics"], str)
+        assert lyrics_song["hasTimestamps"] is False
+
+        # test lyrics with timestamps
+        lyrics_song = yt.get_lyrics(playlist["lyrics"], timestamps = True)
+        assert lyrics_song is not None
+        assert isinstance(lyrics_song["lyrics"], list)
+        assert lyrics_song["hasTimestamps"] is True
+
+        # check the LyricLine object
+        song = lyrics_song["lyrics"][0]
+        assert isinstance(song, LyricLine)
+        assert isinstance(song.text, str)
+        assert isinstance(song.start_time, int)
+        assert isinstance(song.end_time, int)
+        assert isinstance(song.id, int)
```

Comment:
Not sure how useful these asserts are. Better to verify the length of the text or the time being greater than a specific value.

I.e., verify the usefulness of the actual values, not just the type

Reply:
The text could easily be empty, if there was a pause. One character lines are quite common at the beginning of a song (there's that `note character`)